content/dstu2/authorization.md

```diff
@@ -93,45 +93,104 @@ The [bearer access token][7] returned from the authorization server is what you
 
-* grant_type
+* grant_type - "refresh_token"
 * refresh_token
 
 ```
 grant_type=refresh_token&refresh_token={REFRESH_TOKEN}
 ```
 
-If successful, the authorization server will return a **200 OK** response with a content-type of [**application/json**][3] similar to the example below. 
+If successful, the authorization server will return a **200 OK** response with a content-type of [**application/json**][3] similar to the non-normative example below.
 
 ```
 {
-   "access_token":"eyJraWQiOiIyMDE1LTA1LTE1VDE4OjI2OjUxLjM0NiIsInR5cCI6IkpXVCIsImFsZyI6IkVTMjU2In0.eyJzdWIiOiJLTDAyODA2NSIsInVybjpjb206Y2VybmVyOmF1dGhvcml6YXRpb24uY2xhaW1zIjp7InZlciI6IjEuMCIsInRudCI6IjhlMzNhZTNkLTQ4NzItNGQyZi1hMDU5LTVmMjMwN2ZjYmYzYiJ9LCJhenAiOiIyIiwiaXNzIjoiaHR0cDpcL1wvbG9jYWxob3N0OjgwODBcL2F1dGhvcml6YXRpb25cLzhlMzNhZTNkLTQ4NzItNGQyZi1hMDU5LTVmMjMwN2ZjYmYzYlwvdG9rZW4iLCJleHAiOjE0MzE3MTUxNDcsImlhdCI6MTQzMTcxNDU0NywianRpIjoiZmIzZDMwOGQtZmQ0Yy00MTk5LTlmZDQtZTU2ZmEwM2Y3ZTViIn0.Fv_6-LtIFzvf7DHleH-rXsjnaEMFgTHRyok4vJ8hkml5FQtnKejNSwECvqdex7hz6VyclcMy67D_bcafNZjNb",
+   "access_token":"eyJraWQiOiIyMDE2LTAyLTE2VDE2OjM2OjQ4LjY5MiIsInR5cCI6IkpXVCIsImFsZyI6IkVTMjU2In0.eyJzdWIiOiJIRDAxNjUxNCIsInVybjpjb206Y2VybmVyOmF1dGhvcml6YXRpb246Y2xhaW1zIjp7InZlciI6IjEuMCIsInRudCI6IjJjNDAwMDU0LTQyZDgtNGU3NC04N2I3LTgwYjViZDVmZGU5ZiIsImF6cyI6Im9ubGluZV9hY2Nlc3MifSwiYXpwIjoiZGV2anMiLCJpc3MiOiJodHRwczpcL1wvYXV0aG9yaXphdGlvbi5kZXZjZXJuZXIuY29tXC8iLCJleHAiOjE0NTU4OTk2NzAsImlhdCI6MTQ1NTg5OTA3MCwianRpIjoiZWI0Njg1ZDItZGM1MC00Y2NjLWE0YTktYWI1MjJlNzdjZTg1IiwidXJuOmNlcm5lcjphdXRob3JpemF0aW9uOmNsYWltczp2ZXJzaW9uOjEiOnsidmVyIjoiMS4wIiwicHJvZmlsZXMiOnsic21hcnQtdjEiOnsiYXpzIjoib25saW5lX2FjY2VzcyJ9fSwiY2xpZW50IjoiZGV2anMiLCJ1c2VyIjp7InN1YiI6IkhEMDE2NTE0IiwicGVyc29uYSI6InBhdGllbnQiLCJpZHNwIjoiOGUzM2FlM2QtNDg3Mi00ZDJmLWEwNTktNWYyMzA3ZmNiZjNiIiwiaWRzcFVyaSI6Imh0dHBzOlwvXC9hc3NvY2lhdGVzLmRldmNlcm5lci5jb21cL2FjY291bnRzXC9vcGVuaWQifSwidGVuYW50IjoiMmM0MDAwNTQtNDJkOC00ZTc0LTg3YjctODBiNWJkNWZkZTlmIn19.FpGIHS438A9ocU4ozx5s8PzpZ4OhZ6St7S6uP5galSkxrgof07ao1bv2LtFqcbyaFGUTeU3J0NtV82Sfg7GpqA",
    "token_type":"Bearer",
-   "expires_in":600
+   "expires_in":570
 }
 ```
 
-The refresh token issued is good while the user's session is still valid and can be used over and over again until the user's session is no longer valid or the refresh token has been revoked due to being compromised. 
+The refresh token issued is good while the user's session is still valid and can be used over and over again until the user's session is no longer valid or the refresh token has been revoked due to being compromised.
+
+### Error Codes ###
+##### Error URNs for the Authorization Server #####
```

Comment:
So, the actual URL is returned in the OAuth response for an error? Maybe something better would be a recommendation or description of what an app needs to do to support the "standard" error page along with this reference of errors for apps to use for troubleshooting?

Some of these urns would never be returned to the app, so I assume the OAuth server itself is the one that would display the page. EG: unknown-client, invalid-redirect-uri... May need to explain that or indicate which would cause the user to see an error directly rather than the app getting an error?